bucket/lame.json

```
@@ -2,29 +2,30 @@
     "homepage": "https://lame.sourceforge.net/",
     "description": "LAME is a high quality MPEG Audio Layer III (MP3) encoder",
     "license": "LGPL-2.0-only",
-    "version": "3.100",
+    "version": "3.100-20190620",
     "architecture": {
         "64bit": {
-            "url": "http://www.rarewares.org/files/mp3/lame3.100-64.zip",
-            "hash": "c9cddf1650fff0ebe0ba79de794b1fa9b334f43f08e2d365b7a158941ad78fa7"
+            "url": "http://www.rarewares.org/files/mp3/lame3.100-64-20190620.zip",
+            "hash": "d10888ec491a6b7406c357f6825e70ccd1820abcec973c049b7cdfe2a859cbfc"
         },
         "32bit": {
-            "url": "http://www.rarewares.org/files/mp3/lame3.100.zip",
-            "hash": "910e14565e7edc419e3b841d1cd7eb83db52727d0f34fceec2b04d905a148ac6"
+            "url": "http://www.rarewares.org/files/mp3/lame3.100-20190620.zip",
+            "hash": "8779b2de0cde05b7c3a35ff702a169ee086be48d00925e669cdf6fbf87d5619a"
         }
     },
     "bin": "lame.exe",
     "checkver": {
         "url": "http://www.rarewares.org/mp3-lame-bundle.php",
-        "re": "http://www.rarewares.org/files/mp3/lame([\\d.]+).zip"
+        "re": "http://www\\.rarewares\\.org/files/mp3/lame(?<prefix>[\\d.]+)(?<suffix>[^.]*)\\.zip",
```

I would suggest a small tweak to checkver.
```suggestion
        "regex": "/files/mp3/lame(?<prefix>[\\d.]+)-(?<suffix>[^.]*)\\.zip",
        "replace": "${prefix}-{$suffix}"
```

Note: autoupdate.32bit.url = `http://www.rarewares.org/files/mp3/lame$version.zip`
autoupdate.64bit.url = `http://www.rarewares.org/files/mp3/lame$matchPrefix-64-$matchSuffix.zip`